### DISCUSSION TITLE
Angular-datatables Re-Render not working on Angular 17

### DISCUSSION THREAD

**Discussion by campeloneto1** [l-lin/angular-datatables#1790] [General]
Hi, i'm using Angular 17 and i'm trying to re-render some information in my datatables, but it's not working.

This is my code:
https://github.com/campeloneto1/sisac/blob/main/src/app/sistema/views/sexos/sexos.component.html
https://github.com/campeloneto1/sisac/blob/main/src/app/sistema/views/sexos/sexos.component.ts

Can anyone help me?

**Comment by shanmukhateja** [comment]
Hi,

You seem to be using for-loop to render the table body (`tr`).

This practice is not supported anymore.

Please use `TemplateRef` or `Pipe` examples.

I am closing this issue now.
